transmission/transmission.go

```diff
@@ -538,21 +557,39 @@ func (b *batchAgg) enqueueErrResponses(err error, events []*Event, duration time
 	}
 }
 
+var zstdBufferPool sync.Pool
```

Comment:
Where is `sync.Pool` used?

Reply:
zstdBufferPool is used directly below this line - I'm using it to hang onto buffers for the compressed data.

Reply:
:derp: